superset-frontend/src/SqlLab/main.less

```diff
@@ -484,17 +484,18 @@ a.Link {
   margin: 3px 5px;
 }
 
-.tooltip pre {
+.ant-tooltip pre {
```

Comment:
would love to see these styles pulled into an Emotion styles in the component exported in /src/common/components

Reply:
The less LESS we have, the happier I am! Also excited every time we get to see a LESS color variable replaced with an Emotion theme color :D

Reply:
Now these styles are used all over SqlLab, but I wonder if we actually need them for ColumnElement? I think that's the case, so I'm going to make a styled tooltip component only for ColumnElement, but I'd appreciate if you verified that.

Reply:
There is actually 1 case where the global styles affected tooltips outside of ColumnElement - centering of multiline tooltip, such as here:
![image](https://user-images.githubusercontent.com/15073128/99813386-e5064d80-2b47-11eb-92c8-59983d2e7e62.png)
It was centered before, now it's aligned to the left... which I think might be more natural? WDYT @rusackas, @junlincc